kuma/wiki/jinja2/wiki/includes/document_macros.html

```diff
@@ -77,6 +77,12 @@
         <dd class="rev-to"> {{ ', '.join(revision_to.tags.split(',')) }}</dd>
         {% endif %}
 
+        {% if revision_from.comment or revision_to.comment %}
+        <dt>{{ _('Comment:') }}</dt>
```

Comment:
This document has some really creative indenting.

The way @MatonAnthony has indented is consistent with what's around it and aids understanding the code. It's good.